build.xml

```diff
@@ -1483,19 +1517,19 @@
         <jvmarg value="-Dcassandra.test.sstableformatdevelopment=true"/>
         <!-- The first time SecureRandom initializes can be slow if it blocks on /dev/random -->
         <jvmarg value="-Djava.security.egd=file:/dev/urandom" />
-        <jvmarg value="-Dcassandra.testtag=${tag}"/>
+        <jvmarg value="-Dcassandra.testtag=${tag}.jdk${ant.java.version}"/>
```

Comment:
please don't tag it with Java version

Reply:
We do it in a similar way in  `main-5.0` - https://github.com/datastax/cassandra/blob/main-5.0/build.xml#L1190? I find it helpful as we had troubles in the past when things were running in test with wrong JDKs. This makes it explicitly visible in CI.

Reply:
I trim it in Butler anyway; ok